src/features/commands.ts

```diff
@@ -609,15 +609,36 @@ Instead:
 
       await msg.react("⚠️");
 
-      await msg.reply({
-        embed: {
-          title: "Tsk tsk.",
-          type: "rich",
-          description: `Please do **not** try to use \`@here\` or \`@everyone\` - there are ${msg.guild.memberCount} members in Reactiflux. Everybody here is a volunteer, and somebody will respond when they can.`,
-          color: "#BA0C2F",
-        },
+      await msg.channel.messages.fetch({ limit: 10 }).then((messages) => {
```

Comment:
a better option here would be to keep track of the timestamp of the last time it replied with this. and only send the message if that time difference is greater than a given threshold

Reply:
Ok sir I will use that system

Reply:
@ghardin137

Reply:
@ghardin137

Reply:
@ghardin137 but what should be timeout ??? and it should be in  milliseconds, seconds or minutes?????